jwst/resample/resample_spec_step.py

```diff
@@ -206,13 +211,16 @@ def _process_slit(self, input_models):
 
         resamp = resample_spec.ResampleSpecData(input_models, **self.drizpars)
 
-        drizzled_models = resamp.do_drizzle(input_models)
+        library = ModelLibrary(input_models, on_disk=False)
+        library = resamp.do_drizzle(library)
```

Comment:
Is this providing a `ModelLibrary` to `ResampleSpecData`. Based on https://github.com/spacetelescope/jwst/pull/8683/files#r1718909024 and the previous discussion is the plan to update `ResampleSpecData` to handle a `ModelLibrary` or convert it's input to a `ModelContainer`?

Reply:
It's not providing a `ModelLibrary` to as input to `ResampleSpecData`, only to `do_drizzle`, which is shared with the imaging version of `ResampleData`.  The output is a `ModelLibrary`, and is converted back to container in`ResampleSpecStep._process_multislit` and `ResampleSpecStep._process_slit`

Reply:
Thanks! I missed that. So does that mean that with this PR to run `ResampleSpecData.do_drizzle` you need to provide a `ModelContainer` (or list of models) to `ResampleSpecData.__init__` and then a `ModelLibrary` to `do_drizzle`? I'm ok with that, just want to make sure I'm following along.

Reply:
yes that's right